sensor/kubernetes/listener/listener_impl.go

```diff
@@ -82,15 +89,21 @@ func (k *listenerImpl) Stop(_ error) {
 	k.storeProvider.CleanupStores()
 }
 
-func clusterOperatorCRDExists(client client.Interface) (bool, error) {
-	resourceList, err := client.Kubernetes().Discovery().ServerResourcesForGroupVersion(clusterOperatorGroupVersion)
-	if err != nil {
-		return false, err
-	}
-	for _, apiResource := range resourceList.APIResources {
-		if apiResource.Name == clusterOperatorResourceName {
-			return true, nil
+func serverResourcesForGroup(client client.Interface, group string) (*metav1.APIResourceList, error) {
+	resourceList, err := client.Kubernetes().Discovery().ServerResourcesForGroupVersion(group)
+	return resourceList, err
+}
+
+// resourceExists returns true if resource exists in list.  Use with output from
+// `serverResourcesForGroup` to verify a resource exists prior to starting an
+// Informer to prevent client-go from spamming the k8s API and logs.
+func resourceExists(list *metav1.APIResourceList, resource string) bool {
+	for _, apiResource := range list.APIResources {
+		if apiResource.Name == resource {
+			return true
 		}
 	}
-	return false, nil
+
+	log.Warn("Resource %q does not exist...", resource)
```

Comment:
`Warnf`

Reply:
Yup! Linter caught that as well and this has already been updated.